src/components/NavBar.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import '../styles/NavBar.css';
+import PropTypes from 'prop-types';
+
+const NavBar = ({buttons, menuOptions}) => {
+    const [navStatus, useNavBar] = useState(false);
+    return (
+        <>
+            <header className="header">
+                <div className="navContainer">
+                    <span
+                        className="logo"
+                        style={{
+                            color: '#fff',
+                            fontStyle: 'italic',
+                            fontWeight: '400',
+                        }}
+                    >
```

Comment:
all of the implicit returns on this page are also properly formatted, just look weird on this page because they are on one line